components/skeleton/demo/skeleton-group-test-wrapper.js

```diff
@@ -1,7 +1,16 @@
-import { html, LitElement } from 'lit';
+import { css, html, LitElement } from 'lit';
 import { SkeletonGroupMixin } from '../skeleton-group-mixin.js';
 
 class SkeletonGroupTestWrapper extends SkeletonGroupMixin(LitElement) {
+	static get styles() {
+		return css`
+			:host {
```

Comment:
This was defined in the visual-diff HTML files, so just applied it to the demo component instead.